decidim-proposals/spec/features/index_proposal_notes_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Index Proposal Notes", type: :feature do
+  let(:feature) { create(:proposal_feature) }
+  let(:organization) { feature.organization }
+
+  let(:manifest_name) { "proposals" }
+  let(:proposal) { create(:proposal, feature: feature) }
+  let(:participatory_space) { feature.participatory_space }
+
+  let(:body) { "New awesome body" }
+  let(:proposal_notes_count) { 5 }
+
+  let!(:proposal_notes) do
+    create_list(
+      :proposal_note,
+      proposal_notes_count,
+      proposal: proposal
+    )
+  end
+
+  include_context "when managing a feature as an admin"
+
+  before do
+    visit current_path + "proposals/#{proposal.id}/proposal_notes"
+  end
+
+  it "shows all proposal notes for the given proposal" do
```

Comment:
I think this deserves a top context: `when listing all the proposals`, and within that I'd put the `let(:proposal_notes)`. We might have more potential listing scenarios there.